maven-build-caching-samples/enforcer-project/README.md

```diff
@@ -0,0 +1,11 @@
+
+The `maven-enforcer-plugin` cannot be made cacheable in general since it can be extended with rules and we don't know their inputs.
```

Comment:
nice explanation here. will reference this when customers ask.

Reply:
I updated this example so that we now have 4 enforcer rules, 2 of which have been made cacheable